regression/contracts-dfcc/loop_contracts_reject_loops_two_latches/test.desc

```diff
@@ -1,4 +1,4 @@
-CORE dfcc-only
+KNOWNBUG
 main.c
 --dfcc main --apply-loop-contracts
 ^EXIT=10$
```

Comment:
Hmm, no, I think the bug is in this test.desc file, not in the execution of the test: `EXIT=10` is not what it should have been, but it's only the changes in your PR that fix this. `EXIT=0` is correct (and adding `VERIFICATION SUCCESSFUL` may be worthwhile).